src/ParseLiveQuery.js

```diff
@@ -88,6 +88,7 @@ const DefaultLiveQueryController = {
     }
     const currentUser = await CoreManager.getUserController().currentUserAsync();
     const sessionToken = currentUser ? currentUser.getSessionToken() : undefined;
+    const installationId = await CoreManager.getInstallationController().currentInstallationId();
```

Comment:
you should do this async and the current user async together:

```js
const [currentUser, installationId] = await Promise.all([...]);
```

Reply:
Done. I keep forgetting about that trick.